src/validators.ts

```diff
@@ -188,6 +189,43 @@ export function validateMap(format: string, input: string): boolean | string {
   }
 }
 
+/**
+ * Verifies that a file path is relative and does not contain ..
+ * @param input path string to verify
+ */
+export function validateFilePath(input: string): boolean | string {
+  if (input.startsWith('/')) {
+    return 'Please enter a relative path.';
+  }
+
+  if (
+    input
+      .trim()
+      .split('/')
+      .includes('..')
+  ) {
+    return `Please enter a path that does not include '..'.`;
+  }
+
+  return true;
+}
+
+/**
+ * Validates the input using a file format including validating the input path
+ *
+ * @param format Format string wrapped in file()
+ * @param input input to validate
+ */
+export function validateFile(format: string, input: string): boolean | string {
+  try {
+    extractFileFormat(format);
+
+    return validateFilePath(input);
```

Comment:
we'll want a valid extension check to match the format to the end of the filename input